.github/actions/spelling/patterns/patterns.txt

```diff
@@ -22,3 +22,4 @@ Base64::s_(?:En|De)code\(L"[^"]+"
 VERIFY_ARE_EQUAL\(L"[^"]+"
 "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789\+/"
```

Comment:
For reference, the longest word in the dictionary is 46 characters (`pneumonoultramicroscopicsilicovolcanoconiosis`), so check-spelling v0.0.18 will look at words up to 48 characters long. These words are under 28 as check-spelling splits it at `A..Y` / `Za..y`.